doc/technotes/localeModels.rst

```diff
@@ -21,13 +21,14 @@ multiple instances of memories and/or processors with differing
 characteristics.  They may also be hierarchical, with parent locales
 containing one or more child sublocales within them.
 
-There are currently two locale models available, flat and NUMA.  The
+There are currently three locale models available, flat, NUMA, and KNL.  The
 flat model is the default and maps closely to the view of locales
 implemented in the 1.7 release.  The NUMA locale model maps sublocales
 to NUMA domains.  The NUMA model is currently implemented at a prototype
 level. Performance has not yet been a focus in the NUMA locale model and
 will require additional effort in future releases.  We expect to add
-more locale models in future releases.
+more locale models in future releases.  The KNL locale model provides
+access to on-chip high-bandwidth DRAM.
```

Comment:
I agree.  I'm about to push a change that addresses the documentation, which might hide this comment.